demo/mkldnn_quant/quant_aware/PaddleCV_mkldnn_quantaware_tutorial.md

```diff
@@ -0,0 +1,221 @@
+```
+git clone https://github.com/PaddlePaddle/PaddleSlim.git
+cd PaddleSlim
+python setup.py install
+```
+### 1.3 Use paddle and slim in sample code
+You can use paddle and paddleslim as follows:
+```
+import paddle
+import paddle.fluid as fluid
+import paddleslim as slim
+import numpy as np
+```
+
+## 2. Training and converting fp32 model to fp32 qat model
+Users can download pretrained models at [Download pre-trained models](https://github.com/PaddlePaddle/models/blob/develop/PaddleCV/image_classification/README.md)
+
+We provide a script to insert `fake_quantize`/`fake_dequantize` ops, train a few iterations and then save the float32 QAT model. Run as follows
+```
+cd PaddleSlim/demo/mkldnn_quant/quant_aware/
+python train_image_classification.py --model=ResNet50 --pretrained_model=path/to/ResNet50/pretrained --data=imagenet --data_dir=/PATH/TO/ILSVRC2012/  --batch_size=32 --num_epochs=1 --save_float32_qat_dir=/PATH/TO/FLOAT32/QAT/MODEL
+```
+Available options in the above command and their descriptions are as follows:
+- **model:** Model name. Default value: "ResNet50"
+- **pretrained_model:** Path to pre-trained model. Default value: None
+- **data:** Dataset. Default value: imagenet
+- **data_dir:** Path to the dataset. Default value: None
+- **batch_size：** Number of training batch size. Default value: 128
+- **num_epochs:** Number of training epoches. Default value: 1
+- **save_float32_qat_dir:** Path to saved qat_float32 model. Default value: `./quantization_models/`
+- **config_file:** Path to training config file. Default value is `./config.yaml`
+
+If the user needs to change the quantization strategy, modify `config.yaml`. We sugggest the following configuration to obtain the best accuracy.
+```
+config = {
+         'weight_quantize_type': 'channel_wise_abs_max',
+         'activation_quantize_type': 'moving_average_abs_max',
+         'quantize_op_types': ['depthwise_conv2d', 'mul', 'conv2d', 'matmul']
+     }
+```
+**`config.yaml` parameters description：**
+- **quantize_op_types:** Now we support quantizable ops including `depthwise_conv2d`, `mul`, `conv2d`, `matmul`, `transpose2`, `reshape2`, `pool2d`. During the period of training with fake quantize/dequantize op, we only need to insert fake quantize/dequantize ops around 4 types of ops:`depthwise_conv2d`, `mul`, `conv2d`, `matmul`. That is because other quantizable ops `transpose2`, `reshape2`, `pool2d`, input scale and output scale are the same, and the scales could be achieved through `out_threshold` attribute of the op, there is no need to insert fake quantize/dequantize ops for these ops. Hereby, we set `'quantize_op_types': ['depthwise_conv2d', 'mul', 'conv2d', 'matmul']`
+- **Other parameters:** Please refer [PaddleSlim quant_aware API](https://paddlepaddle.github.io/PaddleSlim/api/quantization_api/#quant_aware)
+
+**Note:**
+- To modify the program for quantizaiton, some training options need to disabled. Please disable some options as follows:
```

Comment:
quantizaiton -> quantization
... some training options have to be disabled. Run the following, to disable these options: